Algorithm.Framework/Selection/ManualUniverseSelectionModel.cs

```diff
@@ -39,7 +39,17 @@ public class ManualUniverseSelectionModel : UniverseSelectionModel
         /// Initializes a new instance of the <see cref="ManualUniverseSelectionModel"/> class using the algorithm's
         /// security initializer and universe settings
         /// </summary>
-        /// <param name="symbols">The symbols to subscribe to</param>
+        public ManualUniverseSelectionModel()
```

Comment:
This constructor isn't really needed since there's the `params Symbol[]` constructor which would gladly accept no arguments. That being said, I do like the explicitness of this.